docs/guide-release-testing.md

```diff
@@ -59,17 +59,19 @@ Followed by the options:
   -c, --ciToken                                                         [string]
 ```
 
-Using the `-c <your-token>` option is recommended, as the script will download the required artifacts from Github, instead of having to build from source reducing the testing time. It's suggested to store your [Github Personal Access Token (classic)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#personal-access-tokens-classic) using:
-
-```bash
-export GITHUB_TOKEN=<token>
-```
+Using the `-c <your-token>` option is recommended, as the script will download the required artifacts from Github, instead of having to build from source reducing the testing time.
```

Comment:
```suggestion
Using the `-c <your-token>` option is recommended, as the script will download the required artifacts from GitHub, instead of having to build from source reducing the testing time.
```